src/main/java/com/work/management/service/employee/impl/EmployeeServiceImpl.java

```diff
@@ -108,5 +116,8 @@ public EmployeeDto updateEmployeeEntity(EmployeeDto employeeDto) {
         .collect(toImmutableList());
   }
 
+  private static boolean hasUniqueIds(List<Integer> employeeIds) {
```

Comment:
This function could be inline, as this is a one liner and getting used multiple times, please remove this(inline this).